deb/check_version.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
```

Comment:
dedupe {deb,rpm}/check_version.sh into common/check_version.sh and have a symlink on each with the relative path style.